content/docs/SUSHI/configuration/_index.md

```diff
@@ -50,10 +50,15 @@ SUSHI can generate a simple configuration file for you with the `--init` [option
 #### FSH-Only
 
 If an author wants SUSHI only to build the FHIR definition files, and _not_ to do any additional IG processing, and if the project contains an **ImplementationGuide** resource, then the author does not need to provide a **sushi-config.yaml** at all. If there is no **sushi-config.yaml**, SUSHI will automatically attempt to extract the following information from an **ImplementationGuide** resource:
+* `id`
 * `canonical`
+* `url`
+* `name`
+* `packageId`
```

Comment:
Where are these properties derived from? I tried looking through the code for loading configuration from an IG resource, but I don't see it creating configuration properties for `id`, `url`, `name`, or `packageId`. I do see `parameters` though!

Reply:
Oh wait maybe I understand. This is the list of allowed properties when you use `FSHOnly: true` right? If so, I think the sentence before the list is misleading. That sentence says that this list is the properties extracted from an IG resource, not ones that `sushi-config.yaml` supports. If this is the case, then I think it makes sense to remove `id`, `url`, `name`, and `packageId` from this list, and maybe include a separate list or description about what can be included in a `FSHOnly` `sushi-config.yaml` file.

Reply:
I got these parameters from [the `loadConfigurationFromIgResource` function](https://github.com/FHIR/sushi/blob/888b0c70e57f9a8b0180036ff0e614bb6b6f7297/src/import/loadConfigurationFromIgResource.ts#L62), I think you may have been looking at an older branch? `id`, `url`, `name` and `packageId` were all added fairly recently!

Reply:
Oh.. the irony is that I was on `next`. My bad!